training/culture-and-process/PULL_REQUEST_GUIDANCE.md

```diff
@@ -0,0 +1,35 @@
+Pull Request Guidance
+===========================
+
+General guidance on creating a pull request
+
+### Pre reading
+
+- It is useful to have an understanding of GIT (https://git-scm.com/about) and branches as a way of working
+- Have a look at pull requests in the `dev-code-review` channel on slack
+
+### Pre requisites
+
+None
+
+### Steps
+
+To merge a feature branch into the develop or master/main branch into has to have an approved pull request. 
+
+A pull request can be created in a couple of ways, once the chnages on a local feature branch have been commited and pushed. 
+- In the terminal after pushing a branch a link appears to create a pull request. Follow this url, to create a pull request, but just check the branch (either develop or master/main) that the pull request is being created for.
+- Using the GitHub UI, navigate to the repository and the pull requests tab and use the `new pull request` button on the top right of the screen. 
```

Comment:
This makes sense to me. Thinking about someone who is not familiar with GitHub, is it worth providing a link to the docs? e.g. https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request